audio_capture/src/audio_capture.cpp

```diff
@@ -37,7 +37,10 @@ namespace audio_transport
         std::string device;
         ros::param::param<std::string>("~device", device, "");
 
-        _pub = _nh.advertise<audio_common_msgs::AudioData>("audio", 10, true);
+        std::string topic;
+        ros::param::param<std::string>("~topic", topic, "");
```

Comment:
Same concern here about the default value.

Reply:
I copy pasted the device param and changed it to topic

Reply:
Well that's a fine way to do it, but it could be better. Especially if someone else uses this package without a specified topic.

Anyway the ideal change would be : `ros::param::param<std::string>("~topic", topic, "audio");`